cfme/infrastructure/pxe.py

```diff
@@ -232,7 +233,7 @@ def create(self, cancel=False, refresh=True, refresh_timeout=120):
                                                    'was cancelled by the user')
         else:
             view.add.click()
-            main_view.flash.assert_success_message('PXE Server "{}" was added'.format(self.name))
+            main_view.flash.assert_no_error()
```

Comment:
we just don't need errors.

Reply:
**Question:** I think we still need to check messages for older version than 5.9. And isn't this a bz that we don't have success messages for 5.9?

Reply:
I assume for checking content of messages there're should be separate test. these assertions are not even in tests - but are blocking template creation

Reply:
i'll file BZ on missing messages - maybe there was some error and we should get those messages